Formula/prometheus.rb

```diff
@@ -27,7 +27,10 @@ def caveats; <<~EOS
       #{etc}/prometheus.args
 
     Example configuration:
-      echo "--config.file ~/.config/prometheus.yml" > #{etc}/prometheus.args
+      echo '--config.file $(brew --prefix)/etc/prometheus.yml  \
```

Comment:
This shouldn't be in the caveat if it's already done.

Reply:
Agreed. Removed lines now. and submitted the changes.